atoma-state/src/types.rs

```diff
@@ -258,3 +258,16 @@ pub enum AtomaAtomaStateManagerEvent {
         result_sender: oneshot::Sender<Result<Option<Stack>, AtomaStateManagerError>>,
     },
 }
+
+/// Represents the result of updating the number of compute units in a stack and claiming funds
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
```

Comment:
you should be able to derive `FromRow`

Reply:
Hum, do I want to do that ?

Reply:
if you do that, you don't have to manually fill the struct from the PgRow

Reply:
Addressed